examples/Migration/README.md

```diff
@@ -0,0 +1,19 @@
+# Google Ads Client Library for .NET - Migration examples
+
+This folder contains code examples that illustrate how to migrate from AdWords API to Google Ads API
+ in a step-by-step manner. The following code examples are provided.
+
+## CampaignManagement
+
+This folder contains a code example that shows how to create a Google Ads
+ Search campaign. The code example does the following operations:
+
+  - Create a budget
+  - Create a campaign
+  - Create an ad group.
+  - Create Text Ads
+  - Create keywords
+
+The code examples are numbered from Step 0 to 5, in which we show how to migrate functionality
+ incrementally from AdWords API to Google Ads API. Step 0 example is fully written using
+ AdWords API, and Step 5 example fully uses Google Ads API.
```

Comment:
**the** AdWords API, and by Step 5 the example **has been fully transformed into using the** Google Ads API.

Reply:
Done.